Added the X_FRAME_CONFIGURATION global environment variable and relea… (#3784)
magento/devdocs: guides/v2.1/cloud/env/variables-global.md

  global:
    SKIP_HTML_MINIFICATION: true
```

### `X_FRAME_CONFIGURATION`

-  **Default**—_Not set_
-  **Version**—Magento 2.1.4 and later

> Use the `X_FRAME_CONFIGURATION` variable to change the [`X-Frame-Options`]({{ page.baseurl }}/config-guide/secy/secy-xframe.html) header configuration for the {{ site.data.var.ee }} site. This option determines how the browser can render a page in a `<frame>`, `<iframe>`, or `<object>`:

Reviewer: ```suggestion
Use the `X_FRAME_CONFIGURATION` variable to change the [`X-Frame-Options`]({{ page.baseurl }}/config-guide/secy/secy-xframe.html) header configuration for the {{ site.data.var.ee }} site. Use one of the following options to control how the browser renders a page in a `<frame>`, `<iframe>`, or `<object>`:
```

> - `DENY`: Page cannot be displayed in a frame.

Reviewer: Because these options are caps, there is no need for the code formatting. To be consistent, format it like the True False options in the other variables.

-  **DENY**—Page cannot...

Author: > Because these options are caps, there is no need for the code formatting. To be consistent, format it like the True False options in the other variables.
> 
> * **DENY**—Page cannot...

It was the bolding that was obnoxious.  Removed bolding, add em-dash, and retained code syntax formatting to match other variable formatting.
